components/scream/src/physics/rrtmgp/mo_load_coefficients.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
```

Comment:
I don't think this is part of the C++ standard. It does work in a lot of places, though.

Reply:
Yeah, it's not standard, strictly speaking. I would stick with classic `#ifndef`-like header guards, since that's what happens in all the other parts of scream (and hommexx).

Reply:
Fixed.